feat: sharpen descriptions for five skills that overlap on routing (#93)
NeoLabHQ/context-engineering-kit: plugins/tdd/skills/write-tests/SKILL.md

---
name: write-tests

> description: Add missing test coverage for your local code changes by generating new test files (covers uncommitted and untracked changes, or the latest commit if everything is committed). Use when you want tests written, not a review.

Reviewer: ```suggestion
description: Add missing test coverage for your local code changes by generating new test files (covers uncommitted and untracked changes, or the latest commit if everything is committed). Use when you want write tests for new logic or increase test coverage.
```
